node/cn/backend.go

```diff
@@ -441,6 +442,36 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	cn.addComponent(cn.ChainDB())
 	cn.addComponent(cn.engine)
 
+	// Scan votes for the current epoch. Gov module requires votes for current epoch.
+	// equivalent to `scanAllVotesInEpoch` in headerGovModule.
+	currentEpochStart := currBlock.NumberU64() / pset.Epoch() * pset.Epoch()
+	lowestVoteScannedBlockNumPtr := headergov_impl.ReadLowestVoteScannedBlockNum(cn.chainDB.GetMiscDB())
+	if lowestVoteScannedBlockNumPtr == nil {
+		votes := make(map[uint64]headergov.VoteData)
+		for blockNum := currentEpochStart; blockNum <= cn.blockchain.CurrentBlock().NumberU64(); blockNum++ {
+			header := cn.blockchain.GetHeaderByNumber(blockNum)
+			if len(header.Vote) == 0 {
+				continue
+			}
+
+			vote, err := headergov.VoteBytes(header.Vote).ToVoteData()
+			if err != nil {
+				logger.Error("Error parsing vote", "err", err, "num", blockNum)
+				continue
+			}
+			// TODO-kaiax: consider writing addval/removeval votes to validator DB.
+			if vote != nil && vote.Name() != "governance.addvalidator" && vote.Name() != "governance.removevalidator" {
+				votes[blockNum] = vote
+			}
+		}
```

Comment:
Is it feasible to reuse https://github.com/kaiachain/kaia/pull/113/files#diff-0133e29e4d1ec4ad03f6d391568dcc80f4353a61994822302c651fe528e6de2eR162?

Reply:
I'll move this code to `Start()`

Reply:
Moved to `Init()` d372520